+eui/MappingExpPanel.m

```diff
@@ -19,7 +19,7 @@
     end
   end
   
-  methods %(Access = protected)
+  methods (Access = protected)
```

Comment:
Unrelated, but what is a "mapping" experiment?

Reply:
An old receptive field mapping experiment